pages/stake.tsx

```diff
@@ -194,23 +198,27 @@ const Section = styled.div`
 	flex-direction: column;
 	align-items: center;
 	justify-content: center;
+	margin-top: 1rem;
+	padding: 10px;
+	border-radius: 14px;
+	border: 1px solid ${(props) => props.theme.colors.forestGreen};
 `;
 
```

Comment:
```suggestion
const StyledButton = styled(Button)`
	line-height: 24px;
	font-size: 12px;
	font-family: ${(props) => props.theme.fonts.agrandir};
`;
```

Reply:
We need to use the Typography component for this if it's necessary because the buttons should be same in the whole site.